vision_datasets/common/data_manifest/data_manifest.py

```diff
@@ -85,6 +85,46 @@ def _category_id_check(self, value):
             raise ValueError
 
 
+class MultiImageLabelManifest(ImageLabelManifest):
+    """
+    Encapsulates information of an multi-image label. The label can be associated with a single image, or multi-image jointly. 
+    """
+    def __init__(self,
+                 id: int,
+                 img_ids: List[int],
+                 label_data,
+                 additional_info: Dict = {}):
+        """
+        Annotation manifest organized by img_ids, each set of images can have multiple labels. 
+        Args:
+            id (int): annotation id
+            img_ids: image ids
+            label_data: label data
+            additional_info (dict): additional info about this annotation
+        """
+        self.id = id
+        self.img_ids = img_ids
+        super().__init__(label_data, label_path=None, additional_info=additional_info)
+
+    def __eq__(self, other) -> bool:
+        if not super().__eq__(other):
+            return False
+
+        if not isinstance(other, MultiImageLabelManifest):
+            return False
+
+        return self.id == other.id and self.img_ids == other.img_ids and self.label_data == other.label_data \
+            and self.additional_info == other.additional_info
+
+    def is_negative(self) -> bool:
+        return self.label_data is None
+    
+    def _read_label_data(self):
+        pass
```

Comment:
throws exception maybe, as label_path is always set to None?

Another choice is to support label_path from your ctor, as this seems to be an abstract class without any assumption of the label_data